src/platform/forms-system/test/js/review/ObjectField.unit.spec.jsx

```diff
@@ -31,6 +31,41 @@ describe('Schemaform review: ObjectField', () => {
     expect(tree.getByRole('textbox')).to.exist;
   });
 
+  it('should render with "extra" view:* fields in the ui:order', () => {
+    // When the review ObjectField is rendered, the `view:*` properties are
+    // removed. This test ensures that the `view:*` properties are also removed
+    // from the `ui:order` before running orderProperties(), which will throw an
+    // error if extraneous `ui:order` properties are present (properties which
+    // aren't in the schema).
+    const onChange = sinon.spy();
+    const onBlur = sinon.spy();
+
+    const schema = {
+      properties: {
+        test: {
+          type: 'string',
+        },
+      },
+    };
+
+    const uiSchema = {
+      'ui:order': ['test', 'view:testProp'],
+    };
+
+    const tree = render(
+      <ObjectField
+        schema={schema}
+        uiSchema={uiSchema}
+        idSchema={{}}
+        requiredSchema={{}}
+        onChange={onChange}
+        onBlur={onBlur}
+      />,
+    );
+
+    expect(tree.getByRole('textbox')).to.exist;
```

Comment:
This expectation seems odd to me based on what the test says it is checking for, but I looked at other tests in this file and they are written similarly. I guess this is ok.

Reply:
Yeah, same. :man_shrugging: